NOTES.txt

```diff
@@ -0,0 +1,16 @@
+A template for writing a Configuration (https://docs.crossplane.io/v1.14/concepts/packages).
+
+To use this template:
```

Comment:
this could be updated a bit with the suggestions I had for the README, but feel free to streamline and make it more succinct here :)